### DISCUSSION TITLE
SAML SSO with Google

### DISCUSSION THREAD

**Discussion by devoeger** [kimai/kimai#3269] [Questions & Answers]
I followed the instructions in the documentation.  I get redirected to the google sign in page, select my google account, and get the error "Service is not configured for this user."

From the google SAML app I tried the "test SAML login" button and it gives an "Provided application is not a SAML app" error.

Any suggestions?

**Comment by kevinpapst** [comment]
Can you share what you did exactly:
> I followed the instructions in the documentation

Which documentation?

**Comment by devoeger** [comment]
I follow the instructions here https://www.kimai.org/documentation/saml.html

1.  I modified the local.yaml
   *for connection,idp I changed entityId, sso url, x509cert to match the info from my Google Workspace "SSO with Google as SAML IdP" settings
   *for connection,sp I replaced the 127.0.0.1:8010 with my base url
   *I removed the roles mapping
2.  re-build cache
3. reset permissions
4. tried logging in but received error "Service is not configured for this user."

When this didn't work I tried building a Google custom SAML app using all of the same settings but received "Provided application is not a SAML app".

**Reply by kevinpapst** [reply]
You need to create the Google app from here: https://admin.google.com/ac/apps/unified

**Comment by devoeger** [comment]
Yes, I created an app there that contained all of the same settings as the local.yaml.  When I click the "test SAML login" button from that page it gives the "Provided application is not a SAML app" error.
